src/FacetManipulationTrait.php

```diff
@@ -13,6 +14,13 @@
  */
 trait FacetManipulationTrait {
 
+  /**
+   * The entity field manager
+   *
+   * @var \Drupal\Core\Entity\EntityFieldManager
+   */
+  protected $entityFieldManager;
```

Comment:
Can we keep the typehint here?
```suggestion
  protected EntityFieldManagerInterface $entityFieldManager;
```